boxsdk/object/file.py

```diff
@@ -342,6 +355,10 @@ def update_contents(
             A dictionary containing attributes to add to the file that are not covered by other parameters.
         :type additional_attributes:
             `dict` or None
+        :param sha1:
+            A sha1 checksum for a file.
```

Comment:
should this say for "the new content" (different from the sha1 in the if-match which was the old content)

Reply:
```suggestion
            A sha1 checksum for the new content.
```